plugins/automation/admin/assets/javascripts/admin/components/automation-list.gjs

```diff
@@ -202,10 +204,28 @@ export default class AutomationList extends Component {
                         "discourse_automation.models.automation.enabled.label"
                       }}
                     </div>
-                    <DToggleSwitch
-                      @state={{automation.enabled}}
-                      {{on "click" (fn this.toggleEnabled automation)}}
-                    />
+                    <span class="enabled-toggle-with-tooltip">
+                      {{#if (or automation.enabled automation.canBeEnabled)}}
+                        <DToggleSwitch
+                          @state={{automation.enabled}}
+                          {{on "click" (fn this.toggleEnabled automation)}}
+                        />
+                      {{else}}
+                        <DTooltip @identifier="automation-enabled-toggle">
+                          <:trigger>
+                            <DToggleSwitch
+                              disabled={{true}}
+                              @state={{automation.enabled}}
+                            />
+                          </:trigger>
+                          <:content>
+                            {{i18n
+                              "discourse_automation.models.automation.enable_toggle_disabled"
+                            }}
+                          </:content>
+                        </DTooltip>
```

Comment:
Since this logic is shared between two components, what do you think about extracting this into a component of its own?